src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -56,7 +56,7 @@
 # pylint: disable=line-too-long
 
 
-def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
+def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlation_id=None, https_proxy="", http_proxy="", no_proxy="", proxy_cert="", location=None,
```

Comment:
It seems correlation_id not used in this function.
Can you double check?

Reply:
Hi @wangzelin007 , it is not used in the function, but it is needed as a parameter that should be exposed and it is being set/overridden from a validator. If this parameter is removed from this function, it is not being exposed as a parameter to the "connectedk8s connect" command